src/text.c

```diff
@@ -163,13 +163,12 @@ main(int argc, char *argv[])
 			}
 			break;
 		case 'R':
-			conf->use_netrc = 1;
-			if (optarg) {
-				if (!strlcpy(conf->netrc_filename, optarg,
-				     sizeof(conf->netrc_filename))) {
-					print_help();
-					goto free_conf;
+			{
+				char *netrc_filename = NULL;
+				if (optarg) {
+					netrc_filename = optarg;
 				}
+				conf->netrc = netrc_init(netrc_filename);
```

Comment:
I left it that way because we if we pass `NULL` as the filename (when using the `--netrc` option and _not_ providing a filename), the `netrc.c:netrc_mmap()` function will first check whether the `NETRC` env var exists and will use its content as a the filename. Failing that, it will try to open `$HOME/.netrc`. Thus, I created this variable in order to avoid duplicating the `netrc_init()` call on an `else`. I'll try to refactor it without the var.

Reply:
Done. Solved it with ternary operator.